classes/notification/managerDelegate/EditorialReportNotificationManager.inc.php

```diff
@@ -0,0 +1,223 @@
+<?php
```

Comment:
I watched fast and saw that the other `NotificationManagers` have few code, but I've decided to leave here everything that is connected with the notification...